vite.config.mjs

```diff
@@ -45,11 +44,18 @@ export default defineConfig(({mode}) => {
       sourcemap: true,
       manifest: "manifest.json",
       outDir: './static/',
-      emptyOutDir: true,
+      emptyOutDir: false,
       rollupOptions: {
         input: {
-          'app': 'assets/app.js'
+          'app': 'assets/app/app.js'
         },
+        output: {
+          entryFileNames: (assetInfo) => {
+            console.log(assetInfo);
+            return '[name]-[hash].js';
+          },
+          assetFileNames: "[name]-[hash].[ext]",
```

Comment:
This will flatten the app.js to just /static/app-[hash].js. Otherwise it would get build in `/static/assets/app/` which is weird and won't work.